suite-native/module-send/src/components/SendScreenSubHeader.tsx

```diff
@@ -33,34 +42,50 @@ export const AccountBalanceScreenHeader = ({ accountKey }: AccountBalanceScreenH
         selectAccountAvailableBalance(state, accountKey),
     );
 
+    const tokenSymbol = useSelector((state: TokensRootState) =>
+        selectAccountTokenSymbol(state, accountKey, tokenContract),
+    );
+
+    const tokenBalance = useSelector((state: AccountsRootState) =>
+        selectAccountTokenBalance(state, accountKey, tokenContract),
+    );
+
     if (!networkSymbol) {
         return;
     }
 
+    const accountTitle = tokenSymbol ? `${accountLabel} - ${tokenSymbol}` : accountLabel;
```

Comment:
Can we use en dash (–) instead of hyphen (-) here?

Reply:
fixed [3107f98](https://github.com/trezor/trezor-suite/pull/15099/commits/3107f982ed5688ece658c67db8ed8a4767c62eee)